stl/inc/charconv

```diff
@@ -701,6 +701,8 @@ _NODISCARD inline bool _Multiply(_Big_integer_flt& _Multiplicand, const _Big_int
     return true;
 }
 
+extern const uint32_t _Large_power_data[];

```

Comment:
Now that the declaration and definition are separated, can we provide the exact number of elements?